kotlin-react/src/main/kotlin/react/ComponentType.kt

```diff
@@ -0,0 +1,11 @@
+package react
+
+external interface ComponentType<in P : RProps>
+
+// TODO: Should extend RComponentClassStatics, but has problems with generic params
```

Comment:
Solved?

Reply:
Nope

Reply:
We have discussed that current `RComponentClassStatics` is incorrect, so TODO comment still actual